pkg/apis/toolchain/v1alpha1/cheinstallation_types.go

```diff
@@ -23,6 +23,11 @@ type CheInstallationStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
+	// +optional
+
+	// CheServerURL the URL of the Che Server, once the installation completed
+	// +optional
+	CheServerURL string `json:"CheServerURL,omitempty"`
```

Comment:
It's minor but shouldn't it be
```suggestion
	CheServerURL string `json:"cheServerURL,omitempty"`
```

Reply:
good point! Thanks @alexeykazakov